src/ctapipe/image/tests/test_extractor.py

```diff
@@ -713,7 +717,13 @@ def test_dtype(Extractor, subarray):
     extractor = Extractor(subarray=subarray)
     n_channels, n_pixels, _ = waveforms.shape
     broken_pixels = np.zeros((n_channels, n_pixels), dtype=bool)
+    if Extractor is VarianceExtractor:
```

Comment:
I'd remove this here and just skip the `dl1.image.dtype` check in case it's the `VarianceExtractor`